docs/tutorials/chains/polygon_testnet.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: i18n_page
+title: pages.polygon_testnet
+parent: pages.chains
+grand_parent: pages.tutorials
+nav_order: 1
+---
+
+
+# {%t pages.polygon_testnet %}
+{: .no_toc }
+
+## Table of contents
+{: .no_toc .text-delta }
+
+1. TOC
+{:toc}
+
+---
+
+Starting with FireFly v1.1, it's easy to connect to public Ethereum chains. This guide will walk you through the steps to create a local FireFly development environment and connect it to the public Polygon Mumbai testnet.
+
+## Previous steps: Install the FireFly CLI
+If you haven't set up the FireFly CLI already, please go back to the Getting Started guide and read the section on how to [Install the FireFly CLI](../../gettingstarted/firefly_cli.md).
+
+[← ① Install the FireFly CLI](../../gettingstarted/firefly_cli.md){: .btn .btn-purple .mb-5}
+
+## Create an `evmconnect.yml` config file
+In order to connect to the Polygon testnet, you will need to set a few configuration options for the evmconnect blockchain connector. Create a text file called `evmconnect.yml` with the following contents:
+
+```yml
+confirmations:
+    required: 4
+policyengine.simple:
+    fixedGasPrice: null
+    gasOracle:
+        mode: connector
+```
+
+For this tutorial, we will assume this file is saved at `~/Desktop/evmconnect.yml`. If your path is different, you will need to adjust the path in the next command below.
+
+## Creating a new stack
+To create a local FireFly development stack and connect it to the Polygon Mumbai testnet, we will use command line flags to customize the following settings:
+
+ - Create a new stack named `polygon` with `1` member
+ - Disable `multiparty` mode. We are going to be using this FireFly node as a Web3 gateway, and we don't need to communicate with a consortium here
+ - Connect to an ethereum network
+ - Use the `evmconnect` blockchain connector
+ - Use an remote RPC node. This will create a signer locally, so that our signing key never leaves the development machine.
+ - Set the remote RPC node URL to `https://rpc-mumbai.maticvigil.com`
```

Comment:
Instead of suggesting a specific static endpoint, perhaps we should just point users toward the protocol specific docs. This applies to more than this polygon tutorial.

Perhaps something like:
`See the <protocol docs>and select an HTTPS RPC endpoint`
```
ff init polygon 1 \
    --multiparty=false \
    -b ethereum \
    -c evmconnect \
    -n remote-rpc \
    --remote-node-url <selected rpc endpoint> \
    --chain-id 1287 \
    --connector-config ~/Desktop/evmconnect.yml 
```